go.mod

```diff
@@ -34,6 +34,7 @@ require (
 	github.com/open-telemetry/opentelemetry-collector-contrib/extension/pprofextension v0.84.0
 	github.com/open-telemetry/opentelemetry-collector-contrib/extension/storage v0.84.0
 	github.com/open-telemetry/opentelemetry-collector-contrib/pkg/pdatatest v0.84.0
+  github.com/open-telemetry/opentelemetry-collector-contrib/pkg/stanza v0.84.0
```

Comment:
`make tidy` should format this. We're discussing adding another action to flag go.mod issues.